src/wms/src/main/java/org/geoserver/wms/WMSServiceExceptionHandler.java

```diff
@@ -184,7 +184,7 @@ private void handleImageException(ServiceException exception, Request request, f
         g.setColor(bgcolor);
         g.fillRect(0, 0, img.getWidth(), img.getHeight());
         
-        if (!"BLANK".equals(exceptionFormat)) { //wms 1.3 only
```

Comment:
Okay I got a question on this one - our user manual says:


Format	Syntax	Notes
Blank	EXCEPTIONS=application/vnd.ogc.se_blank		Generates a blank image

It looks like we are writing text onto our blank images - not sure if this is intended.